src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Storage class to handle saving and loading of tasks.
+ */
+public class Storage {
+    /**
+     * File to save and load tasks from.
+     */
+    private File file;
+
+    /**
+     * Constructor for Storage class.
+     * 
+     * @param filePath the path of the file to save and load tasks from.
+     * @throws DukeException
+     */
+    public Storage(String filePath) throws DukeException{
```

Comment:
Is there a missing whitespace between `DukeException` and `{`

Reply:
Good catch!